src/ogpf.f90

```diff
@@ -2248,8 +2248,34 @@ subroutine finalize_plot(this)
         this%hasfileopen = .false.        ! reset file open flag
         this%hasanimation = .false.
         ! Use shell command to run gnuplot
-        call execute_command_line ('gnuplot -persist ' // this%txtfilename)  !   Now plot the results
-
+        if (get_os_type() == 1) then
+            call execute_command_line ('wgnuplot -persist ' // this%txtfilename)  !   Now plot the results

```

Comment:
I set up a judgment (whether the current system is the windows system) to choose which command line to run.

Reply:
Thank you! Sounds good!

Reply:
It seems that the `set term wxt` command in the script file "*.gp" caused the cpu to be occupied when running `gnuplot -persist` under the Windows system.

Thank you for your adoption! 😋@kookma @LKedward 

![set term wxt](https://user-images.githubusercontent.com/32035096/116976954-6a477e00-acf4-11eb-9de2-be1542a92ce2.png)